tests/set-labels/wrong-config-group/.expected/results.yaml

```diff
@@ -8,5 +8,5 @@ items:
     stderr: 'failed to evaluate function: error: function failure'
     exitCode: 1
     results:
-      - message: unknown functionConfig Kind=BadKind ApiVersion=fn.kpt.dev/v1alpha1, expect `SetLabels` or `ConfigMap` with correct formatting
-        severity: error
\ No newline at end of file
+      - message: unknown FunctionConfig `BadKind.fn.kpt.dev`, expect `SetLabels.fn.kpt.dev` or `ConfigMap.v1`
```

Comment:
The new format "KIND.GROUP" is defined from the apimachinery schema which is the libraries the fn/sdk depends on.